arangodb/content.md

```diff
@@ -20,36 +18,29 @@ Joins and Transactions are key features for flexible, secure data designs, widel
 
 Furthermore, ArangoDB offers a microservice framework called [Foxx](https://www.arangodb.com/foxx) to build your own Rest API with a few lines of code.
 
-ArangoDB Documentation
-
--	[ArangoDB Documentation](https://www.arangodb.com/documentation)
--	[ArangoDB Tutorials](https://www.arangodb.com/tutorials)
+ArangoDB Documentation - [ArangoDB Documentation](https://www.arangodb.com/documentation) - [ArangoDB Tutorials](https://www.arangodb.com/tutorials)
 
 ## How to use this image
 
-### Start an `ArangoDB` instance
+### Start an ArangoDB instance
 
 In order to start an ArangoDB instance run
 
-```console
-$ docker run -d --name arangodb-instance arangodb
-```
+	unix> docker run -e ARANGO_RANDOM_ROOT_PASSWORD=1 -d --name arangodb-instance -d arangodb/arangodb
```

Comment:
Indented code blocks vs fenced will lose the color formatting based on the language specifier; either way is fine.

This should just be `arangodb` instead of `arangodb/arangodb`.  (this is applicable throughout the doc)